### ISSUE TITLE
Add capability for vectors and holds in procedures

### ISSUE THREAD

**Issue by erikquinn** [openscope/openscope#20]
SIDs/STARs/IAPs will often include vectors, with the intent of controllers assigning headings to get them to the next phase of their flight. And in Europe, most STARs will actually begin with a holding pattern; if the aircraft does not receive the clearances in time, they will automatically enter the holding pattern.

The ability to add these into procedures in the airport files would enable us to close a big gap in realism with these procedures.

Originally reported in zlsa/atc#439.

---

Blocked by #193.

**Comment by n8rzz** [comment]
@erikquinn this may need to be broken up into two issues.  The holds section of this issue will be completed under #139 ( #193 ).

**Comment by erikquinn** [comment]
Will attempt to cover in #193 (feature #139). If unable, a new issue should be created for the portions not addressed.

Rolled into #139.

**Comment by erikquinn** [comment]
Holds work. Created #310 for the vectors portion of this issue.
